certificate/certificate.go

```diff
@@ -301,6 +321,17 @@ func getExtKeyUsages(cert *x509.Certificate) []string {
 	return usage
 }
 
+func getExtKeyUsageOIDs(cert *x509.Certificate) []string {
+	usage := make([]string, 0)
```

Comment:
doesn't hurt, but also unnecessary. adding `(usage []byte)` as a return in the function signature could get rid of this.